packages/create-vite/template-lit-ts/_gitignore

```diff
@@ -1,5 +1,27 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
 node_modules
 .DS_Store
 dist
-types
+dist-ssr
 *.local
+
+/cypress/videos/
+/cypress/screenshots/
```

Comment:
Thanks for the PR!
But unlike `create-vue`,
- we believe that Vite should be neutral on testing tools
- the templates don't contain a default configuration for Cypress, so the locations for videos and screenshots may change.


So, please remove these Cypress-specific lines from the `gitignore`s.